tests/conftest.py

```diff
@@ -497,3 +498,11 @@ def mock_numtracker_server() -> Iterable[responses.RequestsMock]:
             json=response_with_errors,
         )
         yield requests_mock
+
+
+@pytest.fixture
+def temp_dir() -> Generator[Path]:
```

Comment:
There is already a `directory_path` fixture in `test_scratch` that is exactly the same as this. Could they both use the same one?